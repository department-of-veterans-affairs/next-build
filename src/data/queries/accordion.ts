import { QueryFormatter } from 'next-drupal-query'
import { ParagraphAccordion } from '@/types/drupal/paragraph'
import { Accordion } from '@/types/formatted/accordion'

export const formatter: QueryFormatter<ParagraphAccordion, Accordion> = (
  entity: ParagraphAccordion
) => {
  return {
    id: entity.id,
    header: entity.field_header || null,
    html: entity.field_rich_wysiwyg.processed,
  }
}
