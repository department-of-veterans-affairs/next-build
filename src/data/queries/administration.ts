import { FieldAdministration } from '@/types/drupal/field_type'

export interface Administration {
  entityId: number
  name: string | null
}

/**
 * Formats a Drupal administration field into our standardized Administration interface.
 * 
 * In our Drupal content, the `field_administration` field points to a taxonomy Term
 * that is currently called "Section", but "Section" is only a human label added to
 * it in the Drupal interface, and that name has evolved over time. It was originally,
 * "administration" (and thus the name of the field). Since the only machine name of
 * it is `field_administration`, we'll continue to call it "administration" in our code.
 */
export const formatAdministration = (
  administration: FieldAdministration | null | undefined
): Administration => ({
  entityId: administration?.drupal_internal__tid || null, // "t" is for taxonomy term
  name: administration?.name || null,
})
