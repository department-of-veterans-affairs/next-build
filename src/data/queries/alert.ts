// Define the query params for fetching block--alert.
import { BlockAlert } from '@/types/drupal/block'
import { QueryFormatter } from 'next-drupal-query'
import { Alert } from '@/types/formatted/alert'

export const formatter: QueryFormatter<BlockAlert, Alert> = (
  entity: BlockAlert
) => {
  return {
    alertType:
      entity.field_alert_type === 'information'
        ? 'info'
        : entity.field_alert_type,
    id: entity.id,
    title: entity.field_alert_title,
    content: {
      header: entity.field_alert_content.field_text_expander,
      text: entity.field_alert_content.field_wysiwyg.processed,
    },
  }
}
