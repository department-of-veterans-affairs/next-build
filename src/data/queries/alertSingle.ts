import { QueryFormatter } from 'next-drupal-query'
import { ParagraphAlertSingle } from '@/types/drupal/paragraph'
import {
  AlertSingle,
  AlertBlock,
  AlertNonReusable,
} from '@/types/formatted/alert'
import { formatParagraph } from '@/lib/drupal/paragraphs'
import { queries } from '.'

// TODO:
//
// EDIT:
// It seems *maybe* that next-drupal-query is designed to handle this already.
// You can pass an `id` to queries.getParams().
// E.g.
// queries.getParams('paragraph--alert_single'))
//
// We don't need this here:
// export const params: QueryParams<null> = () => {
//   return queries
//     .getParams()
//     .addInclude([
//       'field_alert_block_reference',
//       'field_alert_block_reference.field_alert_content',
//       'field_alert_non_reusable_ref',
//       'field_alert_non_reusable_ref.field_va_paragraphs',
//     ])
// }
//
// ...but it seems like we might want to define
// paragraph-specific `includes` within the paragraph's
// query file (this file), and then somehow reference these in
// the node query when we need to include fields of a specific
// paragraph type.
//
// E.g.
// export const include = [
//   'field_alert_block_reference',
//   'field_alert_block_reference.field_alert_content',
//   'field_alert_non_reusable_ref',
//   'field_alert_non_reusable_ref.field_va_paragraphs',
// ]
//
// /* Some util file*/
// export const getIncludedSubFields = (
//   parentField: string,
//   subFields: string[]
// ) => [parentField, ...subFields.map((subField) => `${parentField}.${subField}`)]
//
// /* src/data/queries/resourcesSupport.ts */
// import { include } from '@/data/queries/alertSingle'
// export const params: QueryParams<null> = () => {
//   return queries.getParams().addInclude([
//     ...getIncludedSubFields('field_alert_single', include)
//   ])
// }

export const formatter: QueryFormatter<ParagraphAlertSingle, AlertSingle> = (
  entity: ParagraphAlertSingle
) => {
  return {
    type: entity.type as AlertSingle['type'],
    id: entity.id,
    entityId: entity.drupal_internal__id,
    alertSelection:
      entity.field_alert_selection as AlertSingle['alertSelection'],
    blockReference: queries.formatData(
      'block--alert',
      entity.field_alert_block_reference
    ) as AlertBlock,
    nonReusableRef: formatParagraph(
      entity.field_alert_non_reusable_ref
    ) as AlertNonReusable,
  }
}
