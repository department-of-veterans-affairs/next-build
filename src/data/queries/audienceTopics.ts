// Define the query params for fetching node--news_story.
import { ParagraphAudienceTopics } from '@/types/drupal/paragraph'
import { QueryFormatter } from 'next-drupal-query'
import { AudienceTopic, AudienceTopics } from '@/types/formatted/audienceTopics'

const getTagsList = (
  entity: ParagraphAudienceTopics
): AudienceTopic[] | null => {
  if (!entity) return null

  const {
    field_topics: fieldTopics = [],
    field_audience_beneficiares: fieldAudienceBeneficiares,
    field_non_beneficiares: fieldNonBeneficiares,
  } = entity

  const topics = fieldTopics.map((topic) => ({
    id: topic.id,
    href: topic.path?.alias,
    name: topic.name,
    categoryLabel: 'Topics',
  }))

  const audiences = [
    ...(fieldAudienceBeneficiares || []),
    ...(fieldNonBeneficiares || []),
  ]
    .filter((tag) => !!tag)
    .map((audience) => ({
      id: audience.id,
      href: audience.path?.alias,
      name: audience.name,
      categoryLabel: 'Audience',
    }))

  const tagList = [...topics, ...audiences]

  return tagList.sort((a, b) => a.categoryLabel.localeCompare(b.categoryLabel))
}

/**
 * This formatter is unusual in that it takes a paragraph entity and digs into
 * its data to return multiple instances of similar values from it.
 */
export const formatter: QueryFormatter<
  ParagraphAudienceTopics,
  AudienceTopics
> = (entity: ParagraphAudienceTopics) => {
  return {
    type: entity.type as AudienceTopics['type'],
    id: entity.id,
    entityId: entity.drupal_internal__id,
    tags: getTagsList(entity),
  }
}
