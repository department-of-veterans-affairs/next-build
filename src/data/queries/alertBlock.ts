// Define the query params for fetching block--alert.
import { BlockAlert } from '@/types/drupal/block'
import { QueryFormatter } from 'next-drupal-query'
import { AlertBlock, AlertType } from '@/types/formatted/alert'

export const formatter: QueryFormatter<BlockAlert, AlertBlock> = (
  entity: BlockAlert
) => {
  if (!entity) {
    return null
  }

  return {
    alertType: (entity.field_alert_type === 'information'
      ? 'info'
      : entity.field_alert_type) as AlertType,
    id: entity.id,
    title: entity.field_alert_title,
    content: {
      header:
        entity.field_alert_content.field_text_expander ??
        entity.field_alert_heading,
      text: entity.field_alert_content?.field_wysiwyg?.processed ?? '',
    },
  }
}
