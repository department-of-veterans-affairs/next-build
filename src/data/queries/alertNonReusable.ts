// Define the query params for fetching block--alert.
import { BlockAlert } from '@/types/drupal/block'
import { QueryFormatter } from 'next-drupal-query'
import { AlertType, AlertNonReusable } from '@/types/formatted/alert'
import { ParagraphNonReusableAlert } from '@/types/drupal/paragraph'
import { formatParagraph } from '@/lib/drupal/paragraphs'

export const formatter: QueryFormatter<
  ParagraphNonReusableAlert,
  AlertNonReusable
> = (entity: ParagraphNonReusableAlert) => {
  if (!entity) {
    return null
  }

  return {
    type: entity.type as AlertNonReusable['type'],
    id: entity.id,
    entityId: entity.drupal_internal__id,
    alertType: (entity.field_alert_type === 'information'
      ? 'info'
      : entity.field_alert_type) as AlertType,
    heading: entity.field_alert_heading,
    paragraphs: entity.field_va_paragraphs.map(formatParagraph),
  }
}
