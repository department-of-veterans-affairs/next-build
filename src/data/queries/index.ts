import { createQueries } from 'next-drupal-query'
import * as NewsStory from './newsStory'
import * as NewsStoryTeaser from './newsStoryTeaser'
import * as StoryListing from './storyListing'
import * as QuestionAnswer from './questionAnswer'
import * as ExpandableText from './expandableText'
import * as LinkTeaser from './linkTeaser'
import * as MediaImage from './mediaImage'
import * as Banner from './banners'
import * as PersonProfile from './personProfile'
import * as Button from './button'
import * as AudienceTopics from './audienceTopics'
import * as Alert from './alert'
import * as EmailContact from './emailContact'
import * as BenefitsHub from './benefitsHub'
import * as Wysiwyg from './wysiwyg'
import * as StaticPathResources from './staticPathResources'
import * as HeaderFooter from './headerFooter'

export const queries = createQueries({
  // standard Drupal entity data queries
  'node--news_story': NewsStory,
  'node--news_story--teaser': NewsStoryTeaser,
  'node--story_listing': StoryListing,
  'node--q_a': QuestionAnswer,
  'node--person_profile': PersonProfile,
  'node--landing_page': BenefitsHub, // "Benefits Hub Landing Page"
  'paragraph--audience_topics': AudienceTopics,
  'paragraph--button': Button,
  'paragraph--email_contact': EmailContact,
  'paragraph--expandable_text': ExpandableText,
  'paragraph--link_teaser': LinkTeaser,
  'paragraph--rich_text_char_limit_1000': Wysiwyg,
  'paragraph--wysiwyg': Wysiwyg,
  'media--image': MediaImage,
  'block--alert': Alert,

  // Custom queries 
  'banner--alerts_lookup': Banner,
  'header-footer-data': HeaderFooter,

  // Static Path Generation
  'static-path-resources': StaticPathResources,
})
