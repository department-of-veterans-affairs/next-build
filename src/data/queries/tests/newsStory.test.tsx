import { NodeNewsStory } from '@/types/drupal/node'
import { queries } from '@/data/queries'
import mockData from '@/mocks/newsStory.mock.json'

const nodeNewsStoryMock: NodeNewsStory = mockData

describe('node--news_story formatData', () => {
  let windowSpy

  beforeEach(() => {
    windowSpy = jest.spyOn(window, 'window', 'get')
  })

  afterEach(() => {
    windowSpy.mockRestore()
  })

  test('outputs formatted data', () => {
    windowSpy.mockImplementation(() => undefined)

    expect(
      queries.formatData('node--news_story', nodeNewsStoryMock)
    ).toMatchSnapshot()
  })
})
