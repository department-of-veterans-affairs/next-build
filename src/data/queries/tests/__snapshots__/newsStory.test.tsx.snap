// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`node--news_story formatData outputs formatted data 1`] = `
Object {
  "administration": Object {
    "id": 12,
    "name": "VA Pittsburgh health care",
  },
  "author": Object {
    "changed": "2019-08-27T17:50:21+00:00",
    "created": "2019-05-14T16:10:51+00:00",
    "default_langcode": true,
    "drupal_internal__nid": 378,
    "drupal_internal__vid": 9455,
    "field_administration": Object {
      "id": "87832236-1e54-4ce3-8141-8dec27c8a9a7",
      "resourceIdObjMeta": Array [
        null,
      ],
      "type": "taxonomy_term--administration",
    },
    "field_body": null,
    "field_complete_biography": null,
    "field_complete_biography_create": null,
    "field_description": null,
    "field_email_address": null,
    "field_intro_text": null,
    "field_last_name": "Gottschalk",
    "field_media": null,
    "field_meta_tags": null,
    "field_name_first": "Keith",
    "field_office": null,
    "field_phone_number": null,
    "field_photo_allow_hires_download": false,
    "field_suffix": null,
    "id": "c76037be-ebb0-4338-9b31-973a76958929",
    "langcode": "en",
    "links": Object {
      "self": Array [
        null,
      ],
    },
    "metatag": null,
    "moderation_state": "published",
    "node_type": Object {
      "id": "89b07673-7fd5-4292-8fba-58cc10c4e3ec",
      "resourceIdObjMeta": Array [
        null,
      ],
      "type": "node_type--node_type",
    },
    "path": Object {
      "alias": "/pittsburgh-health-care/staff-profiles/keith-gottschalk",
      "langcode": "en",
      "pid": 563,
    },
    "promote": false,
    "relationshipNames": Array [
      "node_type",
      "revision_uid",
      "uid",
      "field_administration",
      "field_complete_biography",
      "field_media",
      "field_office",
    ],
    "resourceIdObjMeta": Object {
      "drupal_internal__target_id": 378,
    },
    "revision_log": "Bulk operation publish revision",
    "revision_timestamp": "2019-08-27T17:50:21+00:00",
    "revision_translation_affected": true,
    "revision_uid": Object {
      "id": "fd71292f-3dc7-409d-b0e8-500204cb005a",
      "resourceIdObjMeta": Array [
        null,
      ],
      "type": "user--user",
    },
    "status": true,
    "sticky": false,
    "title": "Keith Gottschalk",
    "type": "node--person_profile",
    "uid": Object {
      "id": "c0745381-8e20-40da-9540-5a7d2eee1b35",
      "resourceIdObjMeta": Array [
        null,
      ],
      "type": "user--user",
    },
  },
  "bodyContent": "<p>VA Pittsburgh's Outstanding Physicians of the Year for 2019, Drs. Brooke Decker and Aref M. Rahman, stand out for expertly filling critical medical facility roles while fostering a collaborative spirit among staff in caring for Veterans.</p>

<p>Dr. Decker, director of Infection Prevention, sees herself as a problem solver.</p>

<p>\\"The problems that cause infections are incredibly varied and sometimes challenging,\\" said Decker. \\"My job involves getting the right people together to help solve these problems.\\"</p>

<p>Recent examples include ensuring hand sanitizers are plentiful and optimally located to support hand hygiene; safe disposal of needles and syringes; and developing the safest way to shut down and perform maintenance on an air handler in a patient-care area that is always open.</p>

<p>Dr. Rahman, a cardiologist, has been director of the Cardiac Cath Lab for the past six years. He credits his dedicated and skilled team for the lab's success, but he says one other factor plays a big role in that success.</p>


    <p>
    \\"The icing on top is the staff's eagerness to help our Veterans,\\"
    said
    Rahman.
    \\"All hospital environments can be intense and demanding and come with general and specific challenges. But having co-workers whom one can count on makes my job at VA an invaluable opportunity.\\"
    </p>
        n
    

    <p>Decker
    said
    she
    likens
    her
    investigative
    work
    to
    that
    of
    the
    fictional
    private
    detective
    Sherlock
    Holmes
    or
    Dr.
    Gregory
    House,
    a
    fictional
    doctor
    who
    solved
    medical
    mysteries
    on
    television
    's \\"House\\" series.</p>


    <p>
    \\"For every issue, there may be layers and layers of additional clues to unravel,\\"
    said
    Decker.
    \\"The 'aha moment' when it all comes together is very gratifying.\\"
    </p>
        n
    
<p>Even so, Decker says preventing illness in the first place is even more gratifying.</p>


    <p>
    \\"Caring for a single patient and solving that one patient's illness is our honor and privilege as health care providers,\\"
    said
    Decker.
    \\"Preventing 10 patients from getting sick is even better, and that's what we strive to do in Infection Prevention.\\"
    </p>
        n
    
<p>Rahman said he's been fascinated since childhood with how the human body works.</p>


    <p>
    \\"I would read all kinds of articles about new discoveries in medicine and I found myself being drawn more and more toward what is literally the 'heart' of our bodies,\\"
    he
    said.
    \\"Cardiology is not just a career choice, but a calling, and I think it found me.\\"
    </p>
        n
    
<p>Both doctors said the best part of working at VA is listening to Veterans' stories.</p>


    <p>
    \\"I love sitting down at the bedside and hearing about someone's family, pets, adventures,\\"
    said
    Decker.
    \\"I've met some truly great people working at VA.\\"
    </p>
        n
    
<p>Rahman said Veterans' stories make every day at work interesting.</p>


    <p>
    \\"As I interact with my patients, I get to hear a lot of fascinating stories of their experiences all over the world,\\"
    said
    Rahman.
    \\"It's never a dull day at work.\\"
    </p>
        n
    
<p>Decker and Rahman were formally recognized as Outstanding Physicians of the Year on April 26 during the medical staff's quarterly meeting at University Drive.</p>",
  "caption": "
  \\"Caring for a single patient and solving that one patient's illness is our honor and privilege as health care providers.\\"
  -
  Dr.
  Brooke
  Decker
  ",
  "date": "2019-05-14T15:35:12+00:00",
  "entityId": 377,
  "entityPath": "/pittsburgh-health-care/stories/we-honor-outstanding-doctors",
  "id": "bb4dcfc1-736b-4a9e-aa48-23c2dd4a4980",
  "image": Object {
    "alt": null,
    "height": 304,
    "id": "c85b7882-f035-4b55-8b71-fcf81fade350",
    "link": Object {
      "1_1_square_medium_thumbnail": Object {
        "href": "http://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/sites/default/files/styles/1_1_square_medium_thumbnail/public/hub_promos/health-care.png",
        "meta": Object {
          "linkParams": Object {
            "height": "240",
            "rel": Array [
              "drupal://jsonapi/extensions/consumer_image_styles/links/relation-types/#derivative",
            ],
            "width": "240",
          },
          "rel": Array [
            "drupal://jsonapi/extensions/consumer_image_styles/links/relation-types/#derivative",
          ],
        },
      },
      "2_1_large": Object {
        "href": "http://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/sites/default/files/styles/2_1_large/public/hub_promos/health-care.png",
        "meta": Object {
          "linkParams": Object {
            "height": "512",
            "rel": Array [
              "drupal://jsonapi/extensions/consumer_image_styles/links/relation-types/#derivative",
            ],
            "width": "1024",
          },
          "rel": Array [
            "drupal://jsonapi/extensions/consumer_image_styles/links/relation-types/#derivative",
          ],
        },
      },
      "2_1_medium_thumbnail": Object {
        "href": "http://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/sites/default/files/styles/2_1_medium_thumbnail/public/hub_promos/health-care.png",
        "meta": Object {
          "linkParams": Object {
            "height": "240",
            "rel": Array [
              "drupal://jsonapi/extensions/consumer_image_styles/links/relation-types/#derivative",
            ],
            "width": "480",
          },
          "rel": Array [
            "drupal://jsonapi/extensions/consumer_image_styles/links/relation-types/#derivative",
          ],
        },
      },
      "2_3_medium_thumbnail": Object {
        "href": "http://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/sites/default/files/styles/2_3_medium_thumbnail/public/hub_promos/health-care.png",
        "meta": Object {
          "linkParams": Object {
            "height": "480",
            "rel": Array [
              "drupal://jsonapi/extensions/consumer_image_styles/links/relation-types/#derivative",
            ],
            "width": "320",
          },
          "rel": Array [
            "drupal://jsonapi/extensions/consumer_image_styles/links/relation-types/#derivative",
          ],
        },
      },
      "3_2_medium_thumbnail": Object {
        "href": "http://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/sites/default/files/styles/3_2_medium_thumbnail/public/hub_promos/health-care.png",
        "meta": Object {
          "linkParams": Object {
            "height": "320",
            "rel": Array [
              "drupal://jsonapi/extensions/consumer_image_styles/links/relation-types/#derivative",
            ],
            "width": "480",
          },
          "rel": Array [
            "drupal://jsonapi/extensions/consumer_image_styles/links/relation-types/#derivative",
          ],
        },
      },
      "7_2_medium_thumbnail": Object {
        "href": "http://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/sites/default/files/styles/7_2_medium_thumbnail/public/hub_promos/health-care.png",
        "meta": Object {
          "linkParams": Object {
            "height": "300",
            "rel": Array [
              "drupal://jsonapi/extensions/consumer_image_styles/links/relation-types/#derivative",
            ],
            "width": "1050",
          },
          "rel": Array [
            "drupal://jsonapi/extensions/consumer_image_styles/links/relation-types/#derivative",
          ],
        },
      },
      "full_content_width": Object {
        "href": "http://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/sites/default/files/styles/full_content_width/public/hub_promos/health-care.png",
        "meta": Object {
          "linkParams": Object {
            "height": "304",
            "rel": Array [
              "drupal://jsonapi/extensions/consumer_image_styles/links/relation-types/#derivative",
            ],
            "width": "456",
          },
          "rel": Array [
            "drupal://jsonapi/extensions/consumer_image_styles/links/relation-types/#derivative",
          ],
        },
      },
      "large": Object {
        "href": "http://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/sites/default/files/styles/large/public/hub_promos/health-care.png",
        "meta": Object {
          "linkParams": Object {
            "height": "304",
            "rel": Array [
              "drupal://jsonapi/extensions/consumer_image_styles/links/relation-types/#derivative",
            ],
            "width": "456",
          },
          "rel": Array [
            "drupal://jsonapi/extensions/consumer_image_styles/links/relation-types/#derivative",
          ],
        },
      },
      "self": Object {
        "href": "https://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/jsonapi/file/file/e638fd17-9090-442b-a724-5113133b4d0f",
      },
      "viewport_width": Object {
        "href": "http://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/sites/default/files/styles/viewport_width/public/hub_promos/health-care.png",
        "meta": Object {
          "linkParams": Object {
            "height": "304",
            "rel": Array [
              "drupal://jsonapi/extensions/consumer_image_styles/links/relation-types/#derivative",
            ],
            "width": "456",
          },
          "rel": Array [
            "drupal://jsonapi/extensions/consumer_image_styles/links/relation-types/#derivative",
          ],
        },
      },
    },
    "title": null,
    "type": "media--image",
    "url": "http://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/sites/default/files/styles/2_1_large/public/hub_promos/health-care.png",
    "width": 456,
  },
  "introText": "When a hospital has a host of great doctors, honoring just two every year is challenging. ",
  "listing": "/pittsburgh-health-care/stories",
  "published": true,
  "socialLinks": Object {
    "path": "/pittsburgh-health-care/stories/we-honor-outstanding-doctors",
    "title": "We honor outstanding doctors",
  },
  "title": "We honor outstanding doctors",
  "type": "node--news_story",
}
`;
